tools/unreal-vat-demo/Content/Python/build_vat_demo.py

```
@@ -0,0 +1,561 @@
+    if mat is None:
+        unreal.log_error("spawn_dancer_in_level: /Game/VATDemo/M_OpenVAT "
+                         "not found — material build must have failed.")
+        return None
+    unreal.log("spawn_dancer_in_level: loaded SK_Rumba (%s) + M_OpenVAT (%s)"
+               % (mesh, mat))
+
+    # Delete any pre-existing OpenVAT_Dancer so reruns don't pile up.
+    removed = 0
+    try:
+        all_actors = actor_lib.get_all_level_actors()
+    except Exception as e:
```

Ruff (0.15.13)

[warning] Do not catch blind exception: `Exception`

(BLE001)